lib/metadata.js

```diff
@@ -204,9 +209,17 @@ Metadata.prototype.generateOidcPEM = function generateOidcPEM(kid) {
     const exponent = new Buffer(key.e, 'base64');
```

Comment:
keys.some( condition_function ) is equivalent to:

for (key in keys) {
  if condition_function returns true
    break;
  else
   continue;
}

here it's using pubKey as true or false. Yes I should return pubKey, already changed it in the new commit.